volatility3/framework/symbols/windows/extensions/registry.py

```diff
@@ -170,7 +472,7 @@ def _get_subkeys_recursive(
         elif signature == "lh" or signature == "lf":
             listjump = 2
         elif node.vol.type_name.endswith(constants.BANG + "_CM_KEY_NODE"):
-            yield node
+            yield node  # type: ignore
```

Comment:
I just used it there because my language server was complaining about `ObjectInterface` not being an instance of `CM_KEY_NODE`, since it doesn't understand that it's going to have that extension class created instead. I'll remove this and use a `cast` for clarity. We've talked about too much casting being overkill in the past, so if you feel like that's the case here, I'm happy to remove both the cast and the comment.

Reply:
Well, ignoring is definitely bad, but casting feels like somewhat ignoring the problem too, but I guess we'll go with cast.